packages/react-router-dom/__tests__/nav-link-active-test.tsx

```diff
@@ -397,6 +397,33 @@ describe("NavLink", () => {
       expect(anchor.props.className).toMatch("active");
     });
 
```

Comment:
you can eventually add a test to make sure a «Root» Navlink is not always active:


```
<NavLink to="/">Home</NavLink>
…
let anchor = renderer.root.findByType("a");
expect(anchor.props.className).not.toMatch("active");
```

Reply:
I will add the test for this PR but for documentation I will have to create another one.
Thanks for your review!!
Really appreciate that.

Reply:
Added that test `does not apply the default 'active' className to the underlying <a> when at root'/'`